recipes/cysgp4/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "cysgp4" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "24bf9cdc2168c7fc6b6dd89a24849c09560b755d7268e97f6988c08aa09ff84e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
Please remove the `fn` line - it is not required and recommended anymore.

Reply:
Done! Thanks a lot for the review.